src/app/(pages)/(TwoColumn)/article/[id]/page.jsx

```diff
@@ -6,6 +6,7 @@ import TagArea from '@/components/common/TagArea'
 import TagKeyword from '@/components/common/TagKeyword'
 import Feature from '@/components/section/feature/Feature'
 
+export const dynamic = 'force-static';
```

Comment:
@VortexExpansion 
no needed?
Because we defines this website supposed to be generated by server-side (SSG).
Please tell me what cases is this option needed?

Reply:
Checked build again. This is redundant. 
Removed.